pyproject.toml

```diff
@@ -53,13 +53,6 @@ mike = "^1.0.1"
 mkdocs = "^1.2.2"
 mkdocs-material = "^8.1.6"
 python-dotenv = "^0.19.2"
-isort = "^5.10.1"
-mypy = "^0.931"
-black = "^21.10b0"
-pyupgrade = "^2.29.1"
-flake8 = "^4.0.1"
-pre-commit = "^2.15.0"
-flake8-black = "^0.2.3"
```

Comment:
did you do `poetry remove ...`? I assume you still have something in `poetry.lock` file.